tools/logger/log.py

```
@@ -0,0 +1,73 @@
+    def format(self, record: logging.LogRecord):
+        logstr = "[" + datetime.now(self.tz).strftime("%z %Y%m%d %H:%M:%S") + "] ["
+        if self.color:
+            logstr += log_level_color_code.get(record.levelno, colorCodeInfo)
+        logstr += log_level_msg_str.get(record.levelno, record.levelname)
+        if self.color:
+            logstr += colorReset
+        if sys.version_info >= (3, 9):
+            fn = record.filename.removesuffix(".py")
+        elif record.filename.endswith(".py"):
+            fn = record.filename[:-3]
+        logstr += f"] {str(record.name)} | {fn} | {str(record.msg)%record.args}"
```

Ruff (0.14.6)

Use explicit conversion flag

Replace with conversion flag

(RUF010)